src/cvi/modules/common.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+from typing import Callable
 
 class LabelMap():
```

Comment:
maybe add a doc string here

Reply:
Good call; I forgot to docstring some modules/files.